### DISCUSSION TITLE
dual-core 4 thread solution viability

### DISCUSSION THREAD

**Discussion by vanh20** [Security-Onion-Solutions/securityonion#6484] [Unsupported Versions] [answered]
Hello There,

   My i5-6600T system died. It had 32GB memory and a 1TB samsung SSD drive, and it was truly great for my home use case. I'm thinking of replacing the entire unit with a dual-core 4 thread Athlon 3200G and a more recent mainboard that can future proof to a Ryzen cpu and a 64GB ram. This is something I cannot afford at the moment. I know the system requirement asks for [quad processors](https://docs.securityonion.net/en/2.3/hardware.html). But I'm a bit confused as I did certain [comparisons](https://www.cpubenchmark.net/compare/Intel-i5-6600T-vs-AMD-Athlon-3000G-vs-Intel-i5-6400T-vs-Intel-i3-6320-vs-Intel-i3-6098P/2613vs3614vs2668vs2625vs2708) of CPUs with at least one quad underperforming vis-a-vis Athlon 3000G. So my **question to the developers and those that tried it**, Can I trust the cpu mark indicated on the [link](https://www.cpubenchmark.net/compare/Intel-i5-6600T-vs-AMD-Athlon-3000G-vs-Intel-i5-6400T-vs-Intel-i3-6320-vs-Intel-i3-6098P/2613vs3614vs2668vs2625vs2708) (same as the photo) to more or less determine its usability (according to my use case)? **Is dual-core 4 thread solution viable or will it eventually fail when subjected to muti-threading demands of SecurityOnion 2.3?** 

      network traffic: < 40Mbps WAN side
                       = 100Mbps LAN side
      deployment: Standalone, full

   Thank you in advance! :)
   
   
![screenshot-www cpubenchmark net-2021 12 08-17_06_16](https://user-images.githubusercontent.com/79697236/145184004-5acaf0db-1285-4b6c-9021-577acaa5361b.png)

**Comment by vanh20** [comment]
Can I trust the CPU Mark? I believe the answer to that is [YES.](https://www.cpubenchmark.net/cpu_test_info.html) It will definitely underperform compared with my previous i5-6600T, but will do better with the slowest quad core CPUs out there. I know you guys have tried using dual-core 4 threads previously. With enough ram to avoid the use of swap file, does it make it feasible? Any comments will be appreciated. :) Thanks!

**Comment by dougburks** [comment, marked as answer]
Our hardware requirements list 4 CPU cores as the minimum to allow for:
- 1 CPU core for Suricata
- 1 CPU core for Zeek
- 2 CPU cores left over for Elasticsearch, Stenographer, and other processes

For smaller traffic loads like 40Mbps, you may be able to get by with less, particularly if you choose to use Suricata instead of Zeek for metadata.

**Reply by vanh20** [reply]
Thanks for the advice.👍👍
